src/Chapter1.hs

```diff
@@ -620,8 +643,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let last = n `mod` 10
+      sndLast = (n `div` 10) `mod` 10
```

Comment:
Great one!
You can see that this `last1` is actually the `lastDigit` function from the previous task.

Also, I notice that you use

```haskell
mod n 10
```
and 
```haskell
div n 10
```

One hint to make your solution shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!

So you could write it this way:

```haskell
(a, b) = divMod n 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Great suggestion. I will use this. Thank you